templates/ssh_forced_commands_wrapper.py.erb

```diff
@@ -7,14 +7,27 @@ from subprocess import call
 
 
 def check_command():
-    original_command = os.environ['SSH_ORIGINAL_COMMAND']
+    source_command = os.environ.get('zpr_rsync_cmd')
+    if not source_command:
+        sys.exit(1)
+    ssh_command = os.environ['SSH_ORIGINAL_COMMAND']
+    commands_dir = '<%= @permitted_commands %>'
```

Comment:
Sure, though at that point I would rather just hardcode the values. The goal was to make it something that could be expanded upon by a user without having to change the code, but that doesn't matter much now since there aren't really any other users.